elkscmd/bootblocks/boot_sect.S

```diff
@@ -507,14 +503,16 @@ copy_sect:
 //------------------------------------------------------------------------------
 
 msg_boot:
-	.asciz "ELKS boot\r\n"
+	.asciz "ELKS"
 
 msg_error:
 	.asciz "!\r\n"
 
 msg_reboot:
-	.asciz "Press key\r\n"
+	.asciz "Presskey\r\n"
```

Comment:
Beuark 😄 !

Reply:
Oh well... if (and when) this pull request is merged, I will see if I can wring out a few more bytes of space from the FAT boot code... :)

Reply:
@tkchia: Yes - I view it as a stroke of good luck that, thanks to your tight work on the FAT boot loader, that there was any room at all to expand the FAT BPB for compatibility, *and* have the EPB included. The good news is, that boot messages are rarely seen, except in the case of a bad boot. And when they are (I ran into boot problems heavily while developing this PR), they are still very understandable, and provide all the same information required for developer debugging.